source/ProSymbolEditor/Models/SymbolAttributes/LabelAttributes.cs

```diff
@@ -42,47 +42,36 @@ public class LabelAttributes : PropertyChangedBase
         private DomainCodedValuePair _selectedCredibilityDomainPair;
         private string _reliability;
         private DomainCodedValuePair _selectedReliabilityDomainPair;
+        private string _countryCode;
+        private DomainCodedValuePair _selectedCountryCodeDomainPair;
 
         public LabelAttributes() {  }
 
         #region Getters/Setters
-        public DateTime DateTimeValid
+        public DateTime? DateTimeValid
         {
             get
             {
                 return _dateTimeValid;
             }
             set
             {
-                if (value == null)
-                {
-                    _dateTimeValid = DateTime.Now;
-                }
-                else
-                {
-                    _dateTimeValid = value;
-                }
+                _dateTimeValid = value;
 
                 NotifyPropertyChanged(() => DateTimeValid);
             }
         }
 
-        public DateTime DateTimeExpired
+        public DateTime? DateTimeExpired
```

Comment:
I always forget this nullable property syntax - I guess it does have a purpose (like this one)